logging-modules/log4j2-programmatic-configuration/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.baeldung.log4j2</groupId>
+	<artifactId>log4j2-programmatic-configuration</artifactId>
```

Comment:
Let's make sure that this is added to the main build. I think you will need to modify the [main pom](https://github.com/eugenp/tutorials/blob/master/pom.xml). Just grep for `logging-modules`.